packages/beauty-portal-theme/src/components/Chevron/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from 'react';
+
+interface ChevronInterface {
```

Comment:
Other svgs are in images/icons - not sure if we want to do the same here for consistency?

Reply:
Moved into SVG file.